src/views/Home.js

```diff
@@ -28,6 +28,26 @@ export class Home extends Component {
     })
 
   componentDidMount = async () => {
+    if (this.props.navigation.state.params) {
+      console.log(
+        'navigation state params in home :',
+        this.props.navigation.state
+      )
+      const { newCurrentUser } = this.props.navigation.state.params
+      console.log('newCurrentUser : ', newCurrentUser)
+      console.log('CurrentUser in state : ', this.state.currentUser)
+      if (
+        !this.state.currentUser ||
```

Comment:
est-ce que ce n'est pas de trop ?
"!this.state.currentUser"

Reply:
non, parce que au chargement de la page this.state.currentUser = null, et du coup tu prends une erreur en faisant ensuite this.state.currentUser.account sur null